mesonbuild/backend/ninjabackend.py

```diff
@@ -1403,7 +1403,7 @@ def generate_llvm_ir_compile_rule(self, compiler, is_cross, outfile):
     def get_cross_info_lang_args(self, lang, is_cross):
         if is_cross:
             try:
-                return self.environment.cross_info.config['properties'][lang + '_args']
+                return self.environment.cross_info.config['properties']['{!s}_args'.format(lang)]
```

Comment:
Same here. This will just make everything very verbose.

Also `'s'` is the default if you don't specify a modifier.